Can't build ffmpeg 4.3: "vf_blend.asm:416: out of memory"

When trying to build the latest ffmpeg from the homebrew-ffmpeg tap like this:
`brew install homebrew-ffmpeg/ffmpeg/ffmpeg`

I'm getting:

> .....
> X86ASM	libavfilter/x86/vf_blend.o
> CC	libavfilter/x86/vf_blend_init.o
> critical: libavfilter/x86/vf_blend.asm:416: out of memory!
> make: *** [libavfilter/x86/vf_blend.o] Error 2
> make: *** Waiting for unfinished jobs....

Actually I'm using the `--with-fdk-aac` option (that's why I use this tap) but the same error occurs without any options.

It used to work fine before (I've been using this tap since 2019) but not anymore, no idea why. 

I'm using macOS Catalina 10.15.5 (latest version) and already did `brew update`, `brew cleanup`, and `brew doctor`, all OK.

The 'out of memory' is unlikely, I have 32 GB ram, even rebooted my macbook and tried again without running any other applications or background stuff, still the same.

For what it's worth I also tried an alternative tap ([justinmayer/tap](https://github.com/justinmayer/homebrew-tap)) which also offers the `--with-fdk-aac` option, exact same problem occurs there.

It does NOT occur when building the regular ffmpeg with homebrew: just `brew install ffmpeg` works fine. However that lacks the FDK AAC codec which I really need.

Log:

```
2020-06-29 13:01:47 -0700

make
install

GEN	libavutil/libavutil.version
GEN	libswscale/libswscale.version
GEN	libswresample/libswresample.version
GEN	libpostproc/libpostproc.version
GEN	libavcodec/libavcodec.version
GEN	libavformat/libavformat.version
GEN	libavfilter/libavfilter.version
GEN	libavdevice/libavdevice.version
OBJCC	libavdevice/audiotoolbox.o
CC	libavdevice/alldevices.o
CC	libavdevice/avdevice.o
OBJCC	libavdevice/avfoundation.o
CC	libavdevice/caca.o
CXX	libavdevice/decklink_common.o
CXX	libavdevice/decklink_dec.o
CC	libavdevice/decklink_dec_c.o
CXX	libavdevice/decklink_enc.o
CC	libavdevice/decklink_enc_c.o
CC	libavdevice/lavfi.o
CC	libavdevice/reverse.o
CC	libavdevice/sdl2.o
CC	libavdevice/utils.o
CC	libavdevice/xcbgrab.o
CC	libavdevice/xv.o
CC	libavfilter/aeval.o
CC	libavfilter/af_acontrast.o
CC	libavfilter/af_acopy.o
CC	libavfilter/af_acrossover.o
CC	libavfilter/af_acrusher.o
CC	libavfilter/af_adeclick.o
CC	libavfilter/af_adelay.o
CC	libavfilter/af_aderivative.o
CC	libavfilter/af_aecho.o
CC	libavfilter/af_aemphasis.o
CC	libavfilter/af_afade.o
CC	libavfilter/af_afftdn.o
CC	libavfilter/af_afftfilt.o
CC	libavfilter/af_afir.o
CC	libavfilter/af_aformat.o
CC	libavfilter/af_agate.o
CC	libavfilter/af_aiir.o
CC	libavfilter/af_alimiter.o
CC	libavfilter/af_amerge.o
CC	libavfilter/af_amix.o
CC	libavfilter/af_amultiply.o
CC	libavfilter/af_anequalizer.o
CC	libavfilter/af_anlmdn.o
CC	libavfilter/af_anlms.o
CC	libavfilter/af_anull.o
CC	libavfilter/af_apad.o
CC	libavfilter/af_aphaser.o
CC	libavfilter/af_apulsator.o
CC	libavfilter/af_aresample.o
CC	libavfilter/af_arnndn.o
CC	libavfilter/af_asetnsamples.o
CC	libavfilter/af_asetrate.o
CC	libavfilter/af_ashowinfo.o
CC	libavfilter/af_asoftclip.o
CC	libavfilter/af_astats.o
CC	libavfilter/af_asubboost.o
CC	libavfilter/af_atempo.o
CC	libavfilter/af_axcorrelate.o
CC	libavfilter/af_biquads.o
CC	libavfilter/af_bs2b.o
CC	libavfilter/af_channelmap.o
CC	libavfilter/af_channelsplit.o
CC	libavfilter/af_chorus.o
CC	libavfilter/af_compand.o
CC	libavfilter/af_compensationdelay.o
CC	libavfilter/af_crossfeed.o
CC	libavfilter/af_crystalizer.o
CC	libavfilter/af_dcshift.o
CC	libavfilter/af_deesser.o
CC	libavfilter/af_drmeter.o
CC	libavfilter/af_dynaudnorm.o
CC	libavfilter/af_earwax.o
CC	libavfilter/af_extrastereo.o
CC	libavfilter/af_firequalizer.o
CC	libavfilter/af_flanger.o
CC	libavfilter/af_haas.o
CC	libavfilter/af_hdcd.o
CC	libavfilter/af_headphone.o
CC	libavfilter/af_join.o
CC	libavfilter/af_loudnorm.o
CC	libavfilter/af_mcompand.o
CC	libavfilter/af_pan.o
CC	libavfilter/af_replaygain.o
CC	libavfilter/af_rubberband.o
CC	libavfilter/af_sidechaincompress.o
CC	libavfilter/af_silencedetect.o
CC	libavfilter/af_silenceremove.o
CC	libavfilter/af_stereotools.o
CC	libavfilter/af_stereowiden.o
CC	libavfilter/af_superequalizer.o
CC	libavfilter/af_surround.o
CC	libavfilter/af_tremolo.o
CC	libavfilter/af_vibrato.o
CC	libavfilter/af_volume.o
CC	libavfilter/af_volumedetect.o
CC	libavfilter/allfilters.o
CC	libavfilter/asink_anullsink.o
CC	libavfilter/asrc_afirsrc.o
CC	libavfilter/asrc_anoisesrc.o
CC	libavfilter/asrc_anullsrc.o
CC	libavfilter/asrc_hilbert.o
CC	libavfilter/asrc_sinc.o
CC	libavfilter/asrc_sine.o
CC	libavfilter/audio.o
CC	libavfilter/avf_abitscope.o
CC	libavfilter/avf_ahistogram.o
CC	libavfilter/avf_aphasemeter.o
CC	libavfilter/avf_avectorscope.o
CC	libavfilter/avf_concat.o
CC	libavfilter/avf_showcqt.o
CC	libavfilter/avf_showfreqs.o
CC	libavfilter/avf_showspatial.o
CC	libavfilter/avf_showspectrum.o
CC	libavfilter/avf_showvolume.o
CC	libavfilter/avf_showwaves.o
CC	libavfilter/avfilter.o
CC	libavfilter/avfiltergraph.o
CC	libavfilter/bbox.o
CC	libavfilter/boxblur.o
CC	libavfilter/buffersink.o
CC	libavfilter/buffersrc.o
CC	libavfilter/colorspace.o
CC	libavfilter/colorspacedsp.o
CC	libavfilter/dnn/dnn_backend_native.o
CC	libavfilter/dnn/dnn_backend_native_layer_conv2d.o
CC	libavfilter/dnn/dnn_backend_native_layer_depth2space.o
CC	libavfilter/dnn/dnn_backend_native_layer_mathbinary.o
CC	libavfilter/dnn/dnn_backend_native_layer_mathunary.o
CC	libavfilter/dnn/dnn_backend_native_layer_maximum.o
CC	libavfilter/dnn/dnn_backend_native_layer_pad.o
CC	libavfilter/dnn/dnn_backend_native_layers.o
CC	libavfilter/dnn/dnn_interface.o
CC	libavfilter/drawutils.o
CC	libavfilter/ebur128.o
CC	libavfilter/f_bench.o
CC	libavfilter/f_cue.o
CC	libavfilter/f_drawgraph.o
CC	libavfilter/f_ebur128.o
CC	libavfilter/f_graphmonitor.o
CC	libavfilter/f_interleave.o
CC	libavfilter/f_loop.o
CC	libavfilter/f_metadata.o
CC	libavfilter/f_perms.o
CC	libavfilter/f_realtime.o
CC	libavfilter/f_reverse.o
CC	libavfilter/f_select.o
CC	libavfilter/f_sendcmd.o
CC	libavfilter/f_sidedata.o
CC	libavfilter/f_streamselect.o
CC	libavfilter/f_zmq.o
CC	libavfilter/fifo.o
CC	libavfilter/formats.o
CC	libavfilter/framepool.o
CC	libavfilter/framequeue.o
CC	libavfilter/framesync.o
CC	libavfilter/generate_wave_table.o
CC	libavfilter/graphdump.o
CC	libavfilter/graphparser.o
CC	libavfilter/lavfutils.o
CC	libavfilter/log2_tab.o
CC	libavfilter/lswsutils.o
CC	libavfilter/motion_estimation.o
CC	libavfilter/opencl.o
OPENCL	libavfilter/opencl/avgblur.c
OPENCL	libavfilter/opencl/colorkey.c
OPENCL	libavfilter/opencl/colorspace_common.c
OPENCL	libavfilter/opencl/convolution.c
OPENCL	libavfilter/opencl/deshake.c
OPENCL	libavfilter/opencl/neighbor.c
OPENCL	libavfilter/opencl/nlmeans.c
OPENCL	libavfilter/opencl/overlay.c
OPENCL	libavfilter/opencl/pad.c
OPENCL	libavfilter/opencl/tonemap.c
OPENCL	libavfilter/opencl/transpose.c
OPENCL	libavfilter/opencl/unsharp.c
OPENCL	libavfilter/opencl/xfade.c
CC	libavfilter/pthread.o
CC	libavfilter/scale_eval.o
CC	libavfilter/scene_sad.o
CC	libavfilter/setpts.o
CC	libavfilter/settb.o
CC	libavfilter/split.o
CC	libavfilter/src_movie.o
CC	libavfilter/transform.o
CC	libavfilter/trim.o
CC	libavfilter/vaf_spectrumsynth.o
CC	libavfilter/vf_addroi.o
CC	libavfilter/vf_alphamerge.o
CC	libavfilter/vf_amplify.o
CC	libavfilter/vf_aspect.o
CC	libavfilter/vf_atadenoise.o
CC	libavfilter/vf_avgblur.o
CC	libavfilter/vf_avgblur_opencl.o
CC	libavfilter/vf_bbox.o
CC	libavfilter/vf_bilateral.o
CC	libavfilter/vf_bitplanenoise.o
CC	libavfilter/vf_blackdetect.o
CC	libavfilter/vf_blackframe.o
CC	libavfilter/vf_blend.o
CC	libavfilter/vf_bm3d.o
CC	libavfilter/vf_boxblur.o
CC	libavfilter/vf_bwdif.o
CC	libavfilter/vf_cas.o
CC	libavfilter/vf_chromakey.o
CC	libavfilter/vf_chromashift.o
CC	libavfilter/vf_ciescope.o
CC	libavfilter/vf_codecview.o
CC	libavfilter/vf_colorbalance.o
CC	libavfilter/vf_colorchannelmixer.o
CC	libavfilter/vf_colorconstancy.o
CC	libavfilter/vf_colorkey.o
CC	libavfilter/vf_colorkey_opencl.o
CC	libavfilter/vf_colorlevels.o
CC	libavfilter/vf_colormatrix.o
CC	libavfilter/vf_colorspace.o
CC	libavfilter/vf_convolution.o
CC	libavfilter/vf_convolution_opencl.o
CC	libavfilter/vf_convolve.o
CC	libavfilter/vf_copy.o
OBJCC	libavfilter/vf_coreimage.o
CC	libavfilter/vf_cover_rect.o
CC	libavfilter/vf_crop.o
CC	libavfilter/vf_cropdetect.o
CC	libavfilter/vf_curves.o
CC	libavfilter/vf_datascope.o
CC	libavfilter/vf_dblur.o
CC	libavfilter/vf_dctdnoiz.o
CC	libavfilter/vf_deband.o
CC	libavfilter/vf_deblock.o
CC	libavfilter/vf_decimate.o
CC	libavfilter/vf_dedot.o
CC	libavfilter/vf_deflicker.o
CC	libavfilter/vf_dejudder.o
CC	libavfilter/vf_delogo.o
CC	libavfilter/vf_derain.o
CC	libavfilter/vf_deshake.o
CC	libavfilter/vf_deshake_opencl.o
CC	libavfilter/vf_despill.o
CC	libavfilter/vf_detelecine.o
CC	libavfilter/vf_displace.o
CC	libavfilter/vf_dnn_processing.o
CC	libavfilter/vf_drawbox.o
CC	libavfilter/vf_drawtext.o
CC	libavfilter/vf_edgedetect.o
CC	libavfilter/vf_elbg.o
CC	libavfilter/vf_entropy.o
CC	libavfilter/vf_eq.o
CC	libavfilter/vf_extractplanes.o
CC	libavfilter/vf_fade.o
CC	libavfilter/vf_fftdnoiz.o
CC	libavfilter/vf_fftfilt.o
CC	libavfilter/vf_field.o
CC	libavfilter/vf_fieldhint.o
CC	libavfilter/vf_fieldmatch.o
CC	libavfilter/vf_fieldorder.o
CC	libavfilter/vf_fillborders.o
CC	libavfilter/vf_find_rect.o
CC	libavfilter/vf_floodfill.o
CC	libavfilter/vf_format.o
CC	libavfilter/vf_fps.o
CC	libavfilter/vf_framepack.o
CC	libavfilter/vf_framerate.o
CC	libavfilter/vf_framestep.o
CC	libavfilter/vf_freezedetect.o
CC	libavfilter/vf_freezeframes.o
CC	libavfilter/vf_frei0r.o
CC	libavfilter/vf_fspp.o
CC	libavfilter/vf_gblur.o
CC	libavfilter/vf_geq.o
CC	libavfilter/vf_gradfun.o
CC	libavfilter/vf_hflip.o
CC	libavfilter/vf_histeq.o
CC	libavfilter/vf_histogram.o
CC	libavfilter/vf_hqdn3d.o
CC	libavfilter/vf_hqx.o
CC	libavfilter/vf_hue.o
CC	libavfilter/vf_hwdownload.o
CC	libavfilter/vf_hwmap.o
CC	libavfilter/vf_hwupload.o
CC	libavfilter/vf_hysteresis.o
CC	libavfilter/vf_idet.o
CC	libavfilter/vf_il.o
CC	libavfilter/vf_kerndeint.o
CC	libavfilter/vf_lagfun.o
CC	libavfilter/vf_lenscorrection.o
CC	libavfilter/vf_libvmaf.o
CC	libavfilter/vf_limiter.o
CC	libavfilter/vf_lumakey.o
CC	libavfilter/vf_lut.o
CC	libavfilter/vf_lut2.o
CC	libavfilter/vf_lut3d.o
CC	libavfilter/vf_maskedclamp.o
CC	libavfilter/vf_maskedmerge.o
CC	libavfilter/vf_maskedminmax.o
CC	libavfilter/vf_maskedthreshold.o
CC	libavfilter/vf_maskfun.o
CC	libavfilter/vf_mcdeint.o
CC	libavfilter/vf_median.o
CC	libavfilter/vf_mergeplanes.o
CC	libavfilter/vf_mestimate.o
CC	libavfilter/vf_midequalizer.o
CC	libavfilter/vf_minterpolate.o
CC	libavfilter/vf_mix.o
CC	libavfilter/vf_mpdecimate.o
CC	libavfilter/vf_neighbor.o
CC	libavfilter/vf_neighbor_opencl.o
CC	libavfilter/vf_nlmeans.o
CC	libavfilter/vf_nlmeans_opencl.o
CC	libavfilter/vf_nnedi.o
CC	libavfilter/vf_noise.o
CC	libavfilter/vf_normalize.o
CC	libavfilter/vf_null.o
CC	libavfilter/vf_ocr.o
CC	libavfilter/vf_overlay.o
CC	libavfilter/vf_overlay_opencl.o
CC	libavfilter/vf_owdenoise.o
CC	libavfilter/vf_pad.o
CC	libavfilter/vf_pad_opencl.o
CC	libavfilter/vf_palettegen.o
CC	libavfilter/vf_paletteuse.o
CC	libavfilter/vf_perspective.o
CC	libavfilter/vf_phase.o
CC	libavfilter/vf_photosensitivity.o
CC	libavfilter/vf_pixdesctest.o
CC	libavfilter/vf_pp.o
CC	libavfilter/vf_pp7.o
CC	libavfilter/vf_premultiply.o
CC	libavfilter/vf_program_opencl.o
CC	libavfilter/vf_pseudocolor.o
CC	libavfilter/vf_psnr.o
CC	libavfilter/vf_pullup.o
CC	libavfilter/vf_qp.o
CC	libavfilter/vf_random.o
CC	libavfilter/vf_readeia608.o
CC	libavfilter/vf_readvitc.o
CC	libavfilter/vf_remap.o
CC	libavfilter/vf_removegrain.o
CC	libavfilter/vf_removelogo.o
CC	libavfilter/vf_repeatfields.o
CC	libavfilter/vf_rotate.o
CC	libavfilter/vf_sab.o
CC	libavfilter/vf_scale.o
CC	libavfilter/vf_scdet.o
CC	libavfilter/vf_scroll.o
CC	libavfilter/vf_selectivecolor.o
CC	libavfilter/vf_separatefields.o
CC	libavfilter/vf_setparams.o
CC	libavfilter/vf_showinfo.o
CC	libavfilter/vf_showpalette.o
CC	libavfilter/vf_shuffleframes.o
CC	libavfilter/vf_shuffleplanes.o
CC	libavfilter/vf_signalstats.o
CC	libavfilter/vf_signature.o
CC	libavfilter/vf_smartblur.o
CC	libavfilter/vf_spp.o
CC	libavfilter/vf_sr.o
CC	libavfilter/vf_ssim.o
CC	libavfilter/vf_stack.o
CC	libavfilter/vf_stereo3d.o
CC	libavfilter/vf_subtitles.o
CC	libavfilter/vf_super2xsai.o
CC	libavfilter/vf_swaprect.o
CC	libavfilter/vf_swapuv.o
CC	libavfilter/vf_telecine.o
CC	libavfilter/vf_threshold.o
CC	libavfilter/vf_thumbnail.o
CC	libavfilter/vf_tile.o
CC	libavfilter/vf_tinterlace.o
CC	libavfilter/vf_tonemap.o
CC	libavfilter/vf_tonemap_opencl.o
CC	libavfilter/vf_tpad.o
CC	libavfilter/vf_transpose.o
CC	libavfilter/vf_transpose_opencl.o
CC	libavfilter/vf_unsharp.o
CC	libavfilter/vf_unsharp_opencl.o
CC	libavfilter/vf_untile.o
CC	libavfilter/vf_uspp.o
CC	libavfilter/vf_v360.o
CC	libavfilter/vf_vaguedenoiser.o
CC	libavfilter/vf_vectorscope.o
CC	libavfilter/vf_vflip.o
CC	libavfilter/vf_vfrdet.o
CC	libavfilter/vf_vibrance.o
CC	libavfilter/vf_vidstabdetect.o
CC	libavfilter/vf_vidstabtransform.o
CC	libavfilter/vf_vignette.o
CC	libavfilter/vf_vmafmotion.o
CC	libavfilter/vf_w3fdif.o
CC	libavfilter/vf_waveform.o
CC	libavfilter/vf_weave.o
CC	libavfilter/vf_xbr.o
CC	libavfilter/vf_xfade.o
CC	libavfilter/vf_xfade_opencl.o
CC	libavfilter/vf_xmedian.o
CC	libavfilter/vf_yadif.o
CC	libavfilter/vf_yaepblur.o
CC	libavfilter/vf_zoompan.o
CC	libavfilter/vf_zscale.o
CC	libavfilter/video.o
CC	libavfilter/vidstabutils.o
CC	libavfilter/vsink_nullsink.o
CC	libavfilter/vsrc_cellauto.o
CC	libavfilter/vsrc_gradients.o
CC	libavfilter/vsrc_life.o
CC	libavfilter/vsrc_mandelbrot.o
CC	libavfilter/vsrc_mptestsrc.o
CC	libavfilter/vsrc_sierpinski.o
CC	libavfilter/vsrc_testsrc.o
X86ASM	libavfilter/x86/af_afir.o
CC	libavfilter/x86/af_afir_init.o
X86ASM	libavfilter/x86/af_anlmdn.o
CC	libavfilter/x86/af_anlmdn_init.o
X86ASM	libavfilter/x86/af_volume.o
CC	libavfilter/x86/af_volume_init.o
STRIP	libavfilter/x86/af_afir.o
X86ASM	libavfilter/x86/avf_showcqt.o
STRIP	libavfilter/x86/af_anlmdn.o
CC	libavfilter/x86/avf_showcqt_init.o
X86ASM	libavfilter/x86/colorspacedsp.o
CC	libavfilter/x86/colorspacedsp_init.o
X86ASM	libavfilter/x86/scene_sad.o
CC	libavfilter/x86/scene_sad_init.o
X86ASM	libavfilter/x86/vf_atadenoise.o
CC	libavfilter/x86/vf_atadenoise_init.o
X86ASM	libavfilter/x86/vf_blend.o
CC	libavfilter/x86/vf_blend_init.o
critical: libavfilter/x86/vf_blend.asm:416: out of memory!
make: *** [libavfilter/x86/vf_blend.o] Error 2
make: *** Waiting for unfinished jobs....
STRIP	libavfilter/x86/af_volume.o
STRIP	libavfilter/x86/scene_sad.o
STRIP	libavfilter/x86/vf_atadenoise.o
STRIP	libavfilter/x86/avf_showcqt.o
STRIP	libavfilter/x86/colorspacedsp.o

HOMEBREW_VERSION: 2.4.2-55-g84f3dc4
ORIGIN: https://github.com/Homebrew/brew
HEAD: 84f3dc47833a3b469ad815a9f55a00c6f8a15a6e
Last commit: 10 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 9398161363e47d6c6d3920d776e9b0ab8bf78b70
Core tap last commit: 57 minutes ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.EXrSORfQ10/org.macports:0
HOMEBREW_EDITOR: emacs
HOMEBREW_MAKE_JOBS: 16
CPU: 16-core 64-bit kabylake
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
Clang: 11.0 build 1103
Git: 2.27.0 => /usr/local/bin/git
Curl: 7.64.1 => /usr/bin/curl
Java: 16, 15, 14.0.1+8, 14.0.1, 14, 13.0.3, 13.0.2+9, 13.0.2, 12.0.2-zulu-12.3+11, 12.0.2, 11.0.8, 11.0.7+10-LTS, 11.0.7, 11.0.2, 10.0.2-zulu-10.3+5, 10.0.2, 9.0.0-zulu-9.0.7.1, 9, 1.8.0_252-b09, 1.8.0_252, 1.8.0_251, 1.7.0_262
macOS: 10.15.6-x86_64
CLT: 1103.0.32.62
Xcode: 11.6 => /Applications/Xcode-beta.app/Contents/Developer
XQuartz: 2.7.11 => /opt/X11

HOMEBREW_CC: clang
HOMEBREW_CXX: clang++
MAKEFLAGS: -j16
CMAKE_PREFIX_PATH: /usr/local/opt/libffi:/usr/local/opt/openssl@1.1:/usr/local/opt/readline:/usr/local/opt/sqlite:/usr/local/opt/python@3.8:/usr/local/opt/icu4c:/usr/local/opt/libxml2:/usr/local
CMAKE_INCLUDE_PATH: /opt/X11/include:/opt/X11/include/freetype2:/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers
CMAKE_LIBRARY_PATH: /opt/X11/lib:/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries
CMAKE_FRAMEWORK_PATH: /usr/local/opt/python@3.8/Frameworks
PKG_CONFIG_PATH: /usr/local/opt/aom/lib/pkgconfig:/usr/local/opt/dav1d/lib/pkgconfig:/usr/local/opt/libpng/lib/pkgconfig:/usr/local/opt/freetype/lib/pkgconfig:/usr/local/opt/fontconfig/lib/pkgconfig:/usr/local/opt/frei0r/lib/pkgconfig:/usr/local/opt/fribidi/lib/pkgconfig:/usr/local/opt/libffi/lib/pkgconfig:/usr/local/opt/pcre/lib/pkgconfig:/usr/local/opt/openssl@1.1/lib/pkgconfig:/usr/local/opt/readline/lib/pkgconfig:/usr/local/opt/sqlite/lib/pkgconfig:/usr/local/opt/xz/lib/pkgconfig:/usr/local/opt/python@3.8/lib/pkgconfig:/usr/local/opt/glib/lib/pkgconfig:/usr/local/opt/lzo/lib/pkgconfig:/usr/local/opt/pixman/lib/pkgconfig:/usr/local/opt/cairo/lib/pkgconfig:/usr/local/opt/graphite2/lib/pkgconfig:/usr/local/opt/icu4c/lib/pkgconfig:/usr/local/opt/harfbuzz/lib/pkgconfig:/usr/local/opt/libass/lib/pkgconfig:/usr/local/opt/libogg/lib/pkgconfig:/usr/local/opt/libvorbis/lib/pkgconfig:/usr/local/opt/libvpx/lib/pkgconfig:/usr/local/opt/opus/lib/pkgconfig:/usr/local/opt/rtmpdump/lib/pkgconfig:/usr/local/opt/sdl2/lib/pkgconfig:/usr/local/opt/theora/lib/pkgconfig:/usr/local/opt/x264/lib/pkgconfig:/usr/local/opt/x265/lib/pkgconfig:/usr/local/opt/fdk-aac/lib/pkgconfig:/usr/local/opt/game-music-emu/lib/pkgconfig:/usr/local/opt/libbluray/lib/pkgconfig:/usr/local/opt/flac/lib/pkgconfig:/usr/local/opt/libsndfile/lib/pkgconfig:/usr/local/opt/libbs2b/lib/pkgconfig:/usr/local/opt/jpeg/lib/pkgconfig:/usr/local/opt/libtiff/lib/pkgconfig:/usr/local/opt/imlib2/lib/pkgconfig:/usr/local/opt/libcaca/lib/pkgconfig:/usr/local/opt/libmodplug/lib/pkgconfig:/usr/local/opt/gdk-pixbuf/lib/pkgconfig:/usr/local/opt/pango/lib/pkgconfig:/usr/local/opt/librsvg/lib/pkgconfig:/usr/local/opt/libsoxr/lib/pkgconfig:/usr/local/opt/libssh/lib/pkgconfig:/usr/local/opt/libvidstab/lib/pkgconfig:/usr/local/opt/libvmaf/lib/pkgconfig:/usr/local/opt/libxml2/lib/pkgconfig:/usr/local/opt/opencore-amr/lib/pkgconfig:/usr/local/opt/openh264/lib/pkgconfig:/usr/local/opt/little-cms2/lib/pkgconfig:/usr/local/opt/openjpeg/lib/pkgconfig:/usr/local/opt/rav1e/lib/pkgconfig:/usr/local/opt/libsamplerate/lib/pkgconfig:/usr/local/opt/rubberband/lib/pkgconfig:/usr/local/opt/speex/lib/pkgconfig:/usr/local/opt/srt/lib/pkgconfig:/usr/local/opt/webp/lib/pkgconfig:/usr/local/opt/leptonica/lib/pkgconfig:/usr/local/opt/tesseract/lib/pkgconfig:/usr/local/opt/two-lame/lib/pkgconfig:/usr/local/opt/wavpack/lib/pkgconfig:/usr/local/opt/zeromq/lib/pkgconfig:/usr/local/opt/zimg/lib/pkgconfig
PKG_CONFIG_LIBDIR: /usr/lib/pkgconfig:/usr/local/Homebrew/Library/Homebrew/os/mac/pkgconfig/10.15:/opt/X11/lib/pkgconfig:/opt/X11/share/pkgconfig
HOMEBREW_GIT: git
HOMEBREW_SDKROOT: /Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk
ACLOCAL_PATH: /usr/local/opt/libxml2/share/aclocal:/usr/local/share/aclocal:/opt/X11/share/aclocal
PATH: /usr/local/Homebrew/Library/Homebrew/shims/mac/super:/usr/local/opt/nasm/bin:/usr/local/opt/pkg-config/bin:/usr/local/opt/texi2html/bin:/usr/local/opt/aom/bin:/usr/local/opt/dav1d/bin:/usr/local/opt/libpng/bin:/usr/local/opt/freetype/bin:/usr/local/opt/fontconfig/bin:/usr/local/opt/lame/bin:/usr/local/opt/fribidi/bin:/usr/local/opt/gettext/bin:/usr/local/opt/pcre/bin:/usr/local/opt/gdbm/bin:/usr/local/opt/openssl@1.1/bin:/usr/local/opt/sqlite/bin:/usr/local/opt/xz/bin:/usr/local/opt/python@3.8/bin:/usr/local/opt/glib/bin:/usr/local/opt/cairo/bin:/usr/local/opt/graphite2/bin:/usr/local/opt/icu4c/bin:/usr/local/opt/harfbuzz/bin:/usr/local/opt/rtmpdump/bin:/usr/local/opt/sdl2/bin:/usr/local/opt/x264/bin:/usr/local/opt/x265/bin:/usr/local/opt/fdk-aac/bin:/usr/local/opt/libbluray/bin:/usr/local/opt/flac/bin:/usr/local/opt/libsndfile/bin:/usr/local/opt/libbs2b/bin:/usr/local/opt/giflib/bin:/usr/local/opt/jpeg/bin:/usr/local/opt/libtiff/bin:/usr/local/opt/imlib2/bin:/usr/local/opt/libcaca/bin:/usr/local/opt/libgsm/bin:/usr/local/opt/gdk-pixbuf/bin:/usr/local/opt/pango/bin:/usr/local/opt/librsvg/bin:/usr/local/opt/libvmaf/bin:/usr/local/opt/libxml2/bin:/usr/local/opt/little-cms2/bin:/usr/local/opt/openjpeg/bin:/usr/local/opt/rav1e/bin:/usr/local/opt/libsamplerate/bin:/usr/local/opt/rubberband/bin:/usr/local/opt/srt/bin:/usr/local/opt/webp/bin:/usr/local/opt/leptonica/bin:/usr/local/opt/tesseract/bin:/usr/local/opt/two-lame/bin:/usr/local/opt/wavpack/bin:/usr/local/opt/zeromq/bin:/opt/X11/bin:/usr/bin:/bin:/usr/sbin:/sbin
```

i also met the same error, macOS Catalina 10.15.5
```
Last 15 lines from /Users/johan/Library/Logs/Homebrew/ffmpeg/02.make:
STRIP	libavfilter/x86/avf_showcqt.o
CC	libavfilter/x86/avf_showcqt_init.o
X86ASM	libavfilter/x86/colorspacedsp.o
CC	libavfilter/x86/colorspacedsp_init.o
X86ASM	libavfilter/x86/scene_sad.o
CC	libavfilter/x86/scene_sad_init.o
X86ASM	libavfilter/x86/vf_atadenoise.o
STRIP	libavfilter/x86/scene_sad.o
CC	libavfilter/x86/vf_atadenoise_init.o
X86ASM	libavfilter/x86/vf_blend.o
critical: libavfilter/x86/vf_blend.asm:416: out of memory!
make: *** [libavfilter/x86/vf_blend.o] Error 2
make: *** Waiting for unfinished jobs....
STRIP	libavfilter/x86/vf_atadenoise.o
STRIP	libavfilter/x86/colorspacedsp.o

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/homebrew-ffmpeg/homebrew-ffmpeg/issues

These open issues may also help:
Can't build ffmpeg 4.3: "vf_blend.asm:416: out of memory" https://github.com/homebrew-ffmpeg/homebrew-ffmpeg/issues/30
```

I have a similar issue, although it fails on another file:

```
X86ASM	libavcodec/x86/xvididct.o
nasm(84953,0x10f4e65c0) malloc: can't allocate region
*** mach_vm_map(size=140399936016384) failed (error code=3)
nasm(84953,0x10f4e65c0) malloc: *** set a breakpoint in malloc_error_break to debug
critical: libavcodec/x86/xvididct.asm:709: out of memory!
make: *** [libavcodec/x86/xvididct.o] Error 2
```

This is 10.14.6 with latest FFmpeg master, not via Homebrew.

--- a/Formula/ffmpeg.rb
+++ b/Formula/ffmpeg.rb
@@ -28,7 +28,7 @@ class Ffmpeg < Formula
   option "with-libvmaf", "Enable libvmaf scoring library"
   option "with-libxml2", "Enable libxml2 library"
 
-  depends_on "nasm" => :build
+  depends_on "yasm" => :build
   depends_on "pkg-config" => :build
   depends_on "texi2html" => :build
 
@@ -92,6 +92,7 @@ class Ffmpeg < Formula
       --cc=#{ENV.cc}
       --host-cflags=#{ENV.cflags}
       --host-ldflags=#{ENV.ldflags}
+      --x86asmexe=yasm
       --enable-gpl
       --enable-libaom
       --enable-libdav1d

--- a/Formula/ffmpeg.rb
+++ b/Formula/ffmpeg.rb
@@ -28,9 +28,9 @@ class Ffmpeg < Formula
   option "with-libvmaf", "Enable libvmaf scoring library"
   option "with-libxml2", "Enable libxml2 library"
 
-  depends_on "yasm" => :build
   depends_on "pkg-config" => :build
   depends_on "texi2html" => :build
+  depends_on "yasm" => :build
 
   depends_on "aom"
   depends_on "dav1d"

